src/applications/disability-benefits/2346/components/ReviewCardField.jsx

```diff
@@ -284,8 +283,13 @@ class ReviewCardField extends React.Component {
       'vads-u-margin-top--1',
       'vads-u-width--auto',
     ].join(' ');
-
+    const { data } = this.props;
     const { street, city, country } = this.props.formData;
+    const copyOfTempAddress = data.temporaryAddress;
```

Comment:
this isnt necessary as its just a reference to the same place in memory.  You can call ```Object.values(data.temporaryAddress)```